src/sentry/static/sentry/app/views/organizationIntegrations/integrationListDirectory.tsx

```diff
@@ -109,6 +108,8 @@ class OrganizationIntegrations extends AsyncComponent<
       ['publishedApps', '/sentry-apps/', {query: {status: 'published'}}],
       ['appInstalls', `/organizations/${orgId}/sentry-app-installations/`],
       ['plugins', `/organizations/${orgId}/plugins/configs/`],
+      // ['unPublishedApps', '/sentry-apps/', {query: {status: 'unpublished'}}],
+      // ['internalApps', '/sentry-apps/', {query: {status: 'internal'}}],
```

Comment:
@NisanthanNanthakumar shouldn't these be uncommented?

Reply:
or just removed?

Reply:
@scefali I'm going to remove them. I added them to test the endpoints.